app/src/test/java/org/simple/clinic/scanid/ScanSimpleIdEffectHandlerTest.kt

```diff
@@ -203,4 +208,40 @@ class ScanSimpleIdEffectHandlerTest {
     verify(uiActions).openPatientSearch(identifier, null, patientPrefillInfo)
     verifyNoMoreInteractions(uiActions)
   }
+
+  @Test
+  fun `when online patient lookup with identifier is received, then load the online lookup api and patient not found`() {
+    // given
+    val identifier = TestData.identifier(
+        value = "e2696e35-45e9-4982-8555-3de7a7aa4dea",
+        type = BpPassport
+    )
+    val results = LookupPatientOnline.Result.NotFound(identifier = identifier.value)
+    whenever(lookupPatientOnline.lookupWithIdentifier(identifier.value)) doReturn results
+
+    // when
+    testCase.dispatch(OnlinePatientLookupWithIdentifier(identifier))
+
+    // then
+    testCase.assertOutgoingEvents(OnlinePatientLookupWithIdentifierCompleted(results, identifier))
+    verifyNoMoreInteractions(uiActions)
+  }
+
+  @Test
+  fun `when online patient lookup with identifier is received, then load the online lookup api and other other`() {
```

Comment:
Same comment as above, we can rename the test.

`when online patient lookup with identifier effect is received, then lookup patients online with other error`